src/harness/loggedIO.ts

```diff
@@ -367,7 +367,7 @@ namespace Playback {
     function recordReplay<T extends Function>(original: T, underlying: any) {
         function createWrapper(record: T, replay: T): T {
             // tslint:disable-next-line only-arrow-functions
-            return <any>(function () {
+            return <any>(function() {

```

Comment:
these should be done by the formatter. by default we add a space.. but we can remove it using `insertSpaceAfterFunctionKeywordForAnonymousFunctions`.. I do no think we should enable this if it conflicts with our default formatting rules.. it will be a struggle fighting against the editors all the time.

Reply:
#19807